src/main/java/seedu/address/ui/AppointmentCard.java

```diff
@@ -66,31 +56,11 @@ public AppointmentCard(AppointmentView apptView, int displayedIndex) {
         timePeriod.setText(appt.getStartTime() + " - " + appt.getEndTime());
         appointmentType.setText(appt.getAppointmentType().toString());
         note.setText("Notes: " + appt.getNote().toString());
-        Timeline timeline = getTimeline();
-        timeline.play();
-    }
 
-    private Timeline getTimeline() {
-        EventHandler<ActionEvent> appointmentMissedEventHandler = event -> {
-            bindCardPaneStyle();
-        };
-        KeyFrame keyframe = new KeyFrame(Duration.seconds(1), appointmentMissedEventHandler);
-        Timeline timeline = new Timeline(keyframe);
-        timeline.setCycleCount(Animation.INDEFINITE); // Run indefinitely
-        return timeline;
-    }
-
-    private void bindCardPaneStyle() {
-        cardPane.styleProperty().bind(
-                Bindings.when(isBeforeCurrentTime(appt))
-                        .then("-fx-background-color: #FF7074")
-                        .otherwise("-fx-background-color: transparent")
-        );
-    }
+        //make appointment green depending on mark status
 
-    private BooleanBinding isBeforeCurrentTime(Appointment appt) {
-        return Bindings.createBooleanBinding(() ->
-            appt.getStartTime().value.isBefore(LocalTime.now())
-        );
```

Comment:
Hi Xue Ling, this portion of code is important for the implementation for cardPanes to be red when the start time passes the current time, perhaps you accidentally removed them?

Reply:
SORRY MY BAD!! will put it back in 😱